### ISSUE TITLE
MUC Room Issues

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#54]
```
Currently gtalksms/smack throws on exception when it tries to configure the 
room owner. Also the setting publicroom to false has no effect (but at least it 
throws no exception ;)). While I am not yet sure about the cause of this two 
problems, I suggest to make the MUC Server configurable by the user. This would 
have the following benefits:
1. We could easily check if other servers are more suitable/support these XMPP 
Features
2. Improved privacy: The experienced user could choose a server which he trusts 
3. Better response times: The user can choose the same server that also 
provides his account as MUC Server
4. Lower load on jwchat.org, because some users may choose a different server.

I have attached a hg bundle which enables the user to configure the MUC Server.

I hope this will help to find out why the publicroom settings won't work. 
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 19 Jan 2011 at 4:55

Attachments:
- [muc.bundle](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-54/comment-0/muc.bundle)

**Comment by GoogleCodeExporter** [comment]
```
I'll move this preference in xmpp connection part.
I'll add french translations.

```

Original comment by `Florent....@gmail.com` on 19 Jan 2011 at 7:48
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Are you sure that room is still public ? I've done some tests and with this 
option I can't see the room in the public list.
```

Original comment by `Florent....@gmail.com` on 19 Jan 2011 at 7:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 19 Jan 2011 at 8:15
- Changed state: **WaitingForRelease**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Ahh, just thought of the still existing public viewable gtalksms rooms on 
conference.jwchat.org that the setting has now effect. 
Just verified it, it works, my room ist not listed. Sorry for that.

Whats the actual state of the room owners problem? Is it a server or smack 
problem?
```

Original comment by `fschm...@gmail.com` on 19 Jan 2011 at 8:23

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.7.3
```

Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 8:07
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
